deployability/modules/testing/tests/helpers/agent.py

```diff
@@ -444,26 +445,46 @@ def is_agent_port_open(inventory_path):
             str: Os name.
         """
         os_type = HostInformation.get_os_type(inventory_path)
+        time.sleep(5)
         if os_type == 'linux':
-            result = ConnectionManager.execute_commands(inventory_path, 'ss -t -a -n | grep ":1514" | grep ESTAB')
-            if result.get('success'):
-                return 'ESTAB' in result.get('output')
-            else:
-                return False
+            for attempt in range(3):
+                result = ConnectionManager.execute_commands(
+                    inventory_path, 
+                    'ss -t -a -n | grep ":1514" | grep ESTAB'
+                )
+                if result.get('success'):
+                    if 'ESTAB' in result.get('output'):
```

Comment:
Use full status like the other calls

Reply:
ESTAB in Linux Os is the full status message to be asserted.